cla_backend/apps/cla_eventlog/management/commands/find_and_delete_old_cases.py

```diff
@@ -12,33 +13,59 @@ def get_eligible_cases(self):
 
         return Case.objects.filter(created__lte=two_years).exclude(log__created__gte=two_years)
 
+    def get_digital_justice_user_logs(self):
+        return Log.objects.filter(created_by__email__endswith="digital.justice.gov.uk")
+
 
 class Command(BaseCommand):
-    help = (
-        "Find or delete cases that are 2 years old or over that were not deleted prior to the task command being fixed"
-    )
+    help = """
+        Use cases:
+        1. Find or delete cases that are 2 years old or over that were not deleted prior to the task command being fixed
+        2. Delete logs created by users with a @digital.justice.gov.uk email
+        """
+
+    def get_user_input(self, qs_type, qs):
+        return raw_input(
+            "Number of {0} that will be deleted: {1}\nAre you sure about this? (Yes/No) ".format(qs_type, qs.count())
+        )
+
+    def handle_test_command(self, args, cases):
+        if args[0] == "delete":
+            self.instance.run()
+        elif args[0] == "delete-logs":
+            digital_justice_user_logs = self.instance.get_digital_justice_user_logs()
+            self.instance._delete_objects(digital_justice_user_logs)
+
+    def handle_terminal_command(self, args, cases):
+        if args[0] == "delete":
+            if len(args) > 1 and args[1] == "no-input":
+                self.instance.run()
+            else:
+                answer = self.get_user_input("cases", cases)
+                if answer == "Yes":
+                    self.instance.run()
+        elif args[0] == "delete-logs":
+            if len(args) > 1 and args[1] == "no-input":
+                self.instance.run()
```

Comment:
This is wrong. You copy-pasted the condition and didn't change the line that runs

Reply:
Ty 🤦